api/consumer/searchPostDocumentReference/search_post_document_reference.py

```diff
@@ -61,9 +66,20 @@ def handler(
         if body.custodian_identifier
         else None
     )
+    if custodian_id:
+        self_link += f"&custodian:identifier=https://fhir.nhs.uk/Id/ods-organization-code|{custodian_id}"
```

Comment:
Do the contents of the query string need URL encoding? I think there are a few characters in these strings that would usually require encoding, eg: colons, slashes, pipes.

Reply:
After discussion we've decided that the primary purpose these self-links serve atm is to show which search parameters were actually applied (the previous and next page links would be needed for pagination, which is not in scope for MVP; no consumer should need to follow this particular URL as it would take them to the exact result set they are currently reading). It might be more legible without URL encoding. We have agreed to consider URL encoding when pagination links are added to the array.